.github/workflows/build.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       - name: Install rust toolchain
         uses: dtolnay/rust-toolchain@master
         with:
-          toolchain: "1.60"
+          toolchain: "1.63"
```

Comment:
It's a little bit unfortunate to have this crate use a more aggressive MSRV than the Rustls + Webpki crates :-/

The part that has me a little bit confused is that it seems like we're pulling in socket2 from hyper, and hyper's MSRV for the 0.14.x release is lower than 1.60.

Am I missing something?

Reply:
socket2 0.5.3 is being pulled in by Tokio 1.30+. rustls and webpki don't depend on tokio, IIRC? I bumped the tokio-rustls MSRV last week, too: https://github.com/rustls/tokio-rustls/pull/10.

webpki and rustls are just being conservative. That tokio is making the jump might mean that there's not much benefit anymore to keeping the MSRV below 1.63, but IMO the benefit to going to 1.63 is relatively limited for those crates.

Reply:
Makes sense, thanks